apps/verify/behaviours/email-lookup-sender.js

```diff
@@ -18,14 +18,11 @@ const isRecognisedExtension = (email) => {
 };
 
 const checkDomain = (userEmailDomain) => {
-  let flag = false;
-
-  emailDomainList.forEach((emailDomain) => {
-    if (userEmailDomain === emailDomain) {
-      flag = true;
-    }
-  });
-  return flag;
+  const isValid = (emailDomain) => {
+    return (userEmailDomain === emailDomain &&
+      userEmailDomain !== 'gov.uk');
```

Comment:
this shouldn't be needed @cmedlandho it's based on the list
it won't allow a user if it's not on the whitelist